package.json

```diff
@@ -46,7 +46,7 @@
   "keywords": [
     "Scuttling"
   ],
-  "author": "JArnstein",
+  "author": "JArnstein (c) 2015",
```

Comment:
How about:
`"author": "JArnstein © 2015",`

I feel the current implementation is sub-optimal